src/labkey/MultiRequest.ts

```diff
@@ -15,6 +15,74 @@
  */
 import { getOnFailure, getOnSuccess, isArray } from './Utils'
 
+/**
```

Comment:
I built the docs locally and am not seeing your nicely written documentation coming thru for `MultiRequest`. Were you able to verify it?

Reply:
I never quite figured out why this was happening locally, however, with the latest version of TypeDoc things went swimmingly. Going to resolve this as a non-action.